src/app/(evalify)/(academics)/question-bank/page.tsx

```
@@ -83,9 +89,77 @@ interface BankApiResponse {
+  banks: QuestionBank[],
+  searchTerm: string,
+  semesterFilter: number | null,
+): QuestionBank[] => {
+  let filtered = banks;
+
+  if (searchTerm) {
+    filtered = filtered.filter(
+      (bank) =>
+        bank.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+        (bank.courseCode &&
+          bank.courseCode.toLowerCase().includes(searchTerm.toLowerCase())),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)